packages/harmony/src/components/link/TextLink.tsx

```diff
@@ -0,0 +1,72 @@
+import { MouseEvent, useCallback } from 'react'
+
+import { useTheme } from '@emotion/react'
+import { Slot } from '@radix-ui/react-slot'
+
+import { Text } from 'components/text/Text'
+
+import type { TextLinkProps } from './types'
+
+/**
+ * Also known as hyperlinks, these are words or phrases that can be clicked to navigate to a linked webpage.
+ */
+export const TextLink = (props: TextLinkProps) => {
+  const {
+    _isHovered = false,
+    asChild = false,
+    children,
+    variant = 'default',
+    stopPropagation = true,
+    isExternal = false,
+    onClick,
+    ...passthroughProps
+  } = props
+
+  const { color } = useTheme()
+
+  const handleClick = useCallback(
+    (e: MouseEvent<HTMLElement>) => {
+      onClick?.(e)
+      if (stopPropagation) {
+        e.stopPropagation()
```

Comment:
maybe we dont include this option, stopPropagation for now? my guess is that we would add that to the web implementation that uses react-router